### PR TITLE
Users/ksigmund/remove property injection

### PR DESCRIPTION
**Pull request by ksigmund**

### CODE REVIEW

File: src/Microsoft.Sbom.Common/FileSystemUtilsProvider.cs [hunk 1]
```diff
@@ -2,34 +2,24 @@
...
 using System.Runtime.InteropServices;
-using Ninject.Activation;
 
 namespace Microsoft.Sbom.Common
 {
     /// <summary>
     /// Provides the <see cref="IFileSystemUtils"/> for a given OS.
     /// </summary>
-    public class FileSystemUtilsProvider : Provider<IFileSystemUtils>
+    public static class FileSystemUtilsProvider //: Provider<IFileSystemUtils>
```
Comment (ByAgenT) [inline src/Microsoft.Sbom.Common/FileSystemUtilsProvider.cs:11]: nit: remove comment?

File: src/Microsoft.Sbom.Api/Microsoft.Sbom.Api.csproj [hunk 2]
```diff
@@ -47,5 +46,10 @@
       <ProjectReference Include="..\Microsoft.Sbom.Extensions\Microsoft.Sbom.Extensions.csproj" />
       <ProjectReference Include="..\Microsoft.Sbom.Parsers.Spdx22SbomParser\Microsoft.Sbom.Parsers.Spdx22SbomParser.csproj" />
     </ItemGroup>
+  
+    <ItemGroup>
```
Comment (ByAgenT) [inline src/Microsoft.Sbom.Api/Microsoft.Sbom.Api.csproj:50]: What's this ItemGroup does?

**Review by ByAgenT — APPROVED** [review]

**Comment by ByAgenT** [comment]
This is API change, so I think we need to update docs how to use it here https://github.com/microsoft/sbom-tool/blob/main/docs/sbom-tool-api-reference.md

**Comment by ksigmund** [comment]
Abandoning this PR in favor of https://github.com/microsoft/sbom-tool/pull/202

**Pull request closed without merging**